constraint/pkg/client/drivers/local/builtin.go

```diff
@@ -20,7 +21,12 @@ func externalDataBuiltin(d *Driver) func(bctx rego.BuiltinContext, regorequest *
 			return externaldata.HandleError(http.StatusBadRequest, err)
 		}
 
-		externaldataResponse, statusCode, err := d.sendRequestToProvider(bctx.Context, &provider, regoReq.Keys)
+		clientCert, err := tls.LoadX509KeyPair(d.clientCertFile, d.clientKeyFile)
```

Comment:
What happens if the contents of the cert file changes? do we need to watch the file like controller-runtime's webserver does?

Reply:
Since we didn't cache the content, it loads the files every time `external_data` is called so I don't think we need to watch it. Any changes to the files should be picked up by the subsequent call.

Reply:
Ah, I see. Are there concerns about the extra latency this may induce? Disk access can be slow, particularly if its a spinning disk.

Reply:
Let me do some testing on this. I can also introduce a flag to disable this feature by default and user can enable it if they want mTLS between Gateekeeper and providers.

Reply:
+1 to being able to flag-gate this. That'd also be useful for people who are not interested in mTLS.